src/main/java/hudson/plugins/testlink/util/TestLinkHelper.java

```diff
@@ -410,6 +410,10 @@ public static String createReportSummaryDetails(Report report, Report previous)
         return builder.toString();
 	}
 
+	private static String escape(Integer integer) {
+		return integer != null ? Util.escape(integer.toString()) : "null" ;

```

Comment:
Is there a reason to escape integers ? I don't see any case from the `Util.escape` method that could be used when an integer is passed.